envlogger/BUILD.bazel

```diff
@@ -15,9 +15,18 @@
 # Build targets for environment logger.
 load("@py_deps//:requirements.bzl", "requirement")
 load("@rules_python//python:defs.bzl", "py_library")
+load("@rules_python//python:pip.bzl", "compile_pip_requirements")
 
 package(default_visibility = ["//visibility:public"])
 
+# This rule adds a convenient way to update the requirements file.
+compile_pip_requirements(
```

Comment:
If we use `requirement`, then it expects a compiled dependency (generated below).

This is a hack! I couldn't find a good way to create requirements locks with multiple Python versions.

Reply:
I think the rules_python pip repository does not work well with repositories that have native extensions